README.md

```diff
@@ -29,10 +29,12 @@ This repo is composed of 2 projects:
 
 There are very few steps required to create and run your fist unit tests with xUnitRevit:
 
-1. build/install xUnitRevit
-2. create a test library
-3. start Revit, launch the xUnitRevit addin and select the test library
-4. done! Add a star ⭐ to our repo if it was useful 😉
+1. create a copy of the [config sample file](https://github.com/Dre-Tas/xUnitRevit/blob/master/xUnitRevit/config_sample.json) and re-name the copy to `config.json`
```

Comment:
i would make this an relative link rather than an absolute link, but other than that looks good! thanks for adding this. will wait for @teocomi to check and merge 👍

Reply:
Yup as you prefer. No worries with me 😉